projects/igniteui-angular/src/lib/time-picker/time-picker.component.ts

```diff
@@ -408,15 +416,35 @@ export class IgxTimePickerComponent implements
      *<igx-toast #toast message="The time picker has been opened!"></igx-toast>
      * ```
      */
+    @DeprecateProperty(`'onOpen' @Ouput property is deprecated\nUse 'onOpened' instead.`)
     @Output()
     public onOpen = new EventEmitter<IgxTimePickerComponent>();
 
+    /**
+     * Emitted when a timePicker is opened.
+     */
+    @Output()
+    public onOpened = new EventEmitter<IgxTimePickerComponent>();
+
     /**
      * Emitted when a timePicker is being closed.
      */
+    @DeprecateProperty(`'onClose' @Ouput property is deprecated\nUse 'onClosed' instead.`)
```

Comment:
Typo `@Ouput`. Replace `\n` with `.` so there won't be a new line in the output.